packages/webpack/example/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "example",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "open ./dist/index.html",
```

Comment:
Is this file generated? Or should this line be removed from here?

Reply:
The webpack example project puts an `index.html` directly in the `dist` folder, but I'll go ahead and move it up to the root since we don't want to check in `dist`.